### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by mjauvin**
- add documentation for media_path() helper method
- add "svg" to the image validation type extensions.

### CODE REVIEW

File: services-validation.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ Passing `NULL` as a "where" clause value will add a check for a `NULL` database
 <a name="rule-image"></a>
 #### image
 
-The file under validation must be an image (jpeg, png, bmp, or gif)
+The file under validation must be an image (jpeg, png, bmp, svg or gif)
```
Comment (LukeTowers) [inline services-validation.md:312]: Is this correct? I thought October still doesn't allow SVGs unless explicitly enabled in the config.

Comment (mjauvin) [reply services-validation.md:312]: The validation itself comes from laravel and svg are allowed

**Comment by mjauvin** [comment]
Hey @bennothommo, why did you mark this as blocked?

**Comment by LukeTowers** [comment]
@mjauvin waiting on unit tests in the related PR would be my guess.

**Comment by mjauvin** [comment]
right, forgot this was also updating docs for the other PR... thought this was only for the svg thingy...

**Comment by mjauvin** [comment]
Not relevant anymore.

**Pull request closed without merging**